app/Repositories/UserRepository.php

```diff
@@ -112,6 +113,23 @@ public function counts()
     }
 
     /**
+     * Get blog authors, their number of blogs, the last created date, and title of their latest blog entry.
+     *
+     * @return Illuminate\Database\Eloquent\Builder
+     */
+    public function getAuthorPostsWithOrder()
+    {
+        $authors = $this->model
+            ->join('posts', 'posts.user_id', '=', 'users.id')
+            ->select(DB::raw('count(posts.id) as total_count'), 'username', 'posts.id', 'posts.created_at', 'posts.title')
```

Comment:
I see.. https://blog.jetbrains.com/phpstorm/2015/01/laravel-development-using-phpstorm/ IntelliJ needs some help for Laravel facades..